docs/adr/0024-use-travis-for-continuous-integration.md

```diff
@@ -0,0 +1,44 @@
+# Use TravisCI for Continuous Integration
+
+* Status: Accepted
+* Date: 2018-06-27
+
+## Context and Problem Statement
+
+Winery needs to be build using continuous integration.
+
+## Decision Drivers
+
+* Easy to configure CI solution
+* For free CI solution
+* Only important output should be logged
+
+## Considered Options
+
+* [TravisCI](https://docs.travis-ci.com/)
+* [CircleCI](https://circleci.com/docs/2.0/)
+* [Eclipse Common Build Infrastructure - Jenkins](https://wiki.eclipse.org/Jenkins)
+
+## Decision Outcome
+
+Chosen option: "CircleCI", because
```

Comment:
Please adjust the ADR and reference https://bugs.eclipse.org/bugs/show_bug.cgi?id=536180